website-scraper/index.d.ts

```diff
@@ -0,0 +1,48 @@
+// Type definitions for website-scraper v1.2.x
+// Project: https://github.com/s0ph1e/node-website-scraper
+// Definitions by: Christian Rackerseder <https://www.echooff.de>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module "website-scraper" {
```

Comment:
Hmm what exactly do you mean? I don't quite understand your question 🤔 
Do you mean that I don't have to write a `declare module` and should export the function directly?